keps/sig-api-machinery/0006-apply.md

```diff
@@ -163,6 +163,9 @@ Kubernetes clusters. The semantical differences between server-side
 apply and client-side apply will make a smooth roll-out difficult, so
 the best way to achieve this has not been decided yet.
 
+A new update to the implementation of server-side apply prevents objects with managed
```

Comment:
Phrasing should probably not be "A new update..." but more "Applied objects can not contain ...". Git is maintaining the history, it doesn't have to be in text.

Also I don't think this should be in kubectl since it's a feature of the apiserver.

Reply:
It should still be in this file right?